web-common/src/features/dashboards/time-dimension-details/TDDTable.svelte

```diff
@@ -65,9 +65,9 @@
       "bg-white",
       "bg-gray-100",
       "bg-gray-200",
-      "bg-blue-50",
-      "bg-blue-100",
-      "bg-blue-200",
+      "bg-primary-50",
+      "bg-primary-100",
+      "bg-primary-200",
       "bg-slate-50",
       "bg-slate-100",
       "bg-slate-200",
```

Comment:
@jkhwu and @djbarnwal  would we want to migrate these to be `muted` rather than `slate`

Reply:
I'm unable to interpret these colors in context so I might need more info to respond here. Generally in the app I intend to keep slate where I've used slate before.